### PR TITLE
[Task #68295 #68299] Edit Salon

### PR DESCRIPTION
**Pull request by Hungnd-0370**
## Related Tickets:
- [#68295](https://edu-redmine.sun-asterisk.vn/issues/68295) 
- [#68299](https://edu-redmine.sun-asterisk.vn/issues/68299)

## WHAT this PR does?
- Edit salon 

## Notes Impacted Areas:
- N/A.

## Attachments:
- 
image

### CODE REVIEW

**Comment by Hungnd-0370** [comment]
ready

File: resources/js/Pages/salons/Edit.jsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                    <h2 className='font-semibold text-2xl text-gray-800 leading-tight'>{lang.get('strings.Edit-Salon')}</h2>
+                </div> 
+                <div className="bg-white w-full shadow overflow-hidden pt-10 pb-4 border sm:rounded-lg">
+                    <Form
+                        labelCol={{
+                        span: 4,
+                        }}
+                        wrapperCol={{
+                        span: 14,
+                        }}
```
Comment (ghost) [inline resources/js/Pages/salons/Edit.jsx]:
```suggestion
                        labelCol={{
                            span: 4,
                        }}
                        wrapperCol={{
                            span: 14,
                        }}
```

File: resources/js/Pages/salons/Edit.jsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                        >
+                            <Input disabled={true}/>
+                        </Form.Item>
+                        <Form.Item 
+                            label={lang.get('strings.Name')}
+                            name = "name"
+                            rules={[
+                                {
+                                  required: true,
+                                  message: 'Please input shop\'s name!',
```
Comment (ghost) [inline resources/js/Pages/salons/Edit.jsx]: localization =))

**Comment by Hungnd-0370** [comment]
ready

**Comment by Hungnd-0370** [comment]
ready

**Pull request merged**